ct-aa/dotTheory---A11y
# FE-1005: Added the sample pages with form for the A11y training

# Related Tickets
[FE-1005](https://codeandtheory.atlassian.net/browse/FE-1005)

# Description
- Added the sample page for the A11y form exercise.
- Added the sample page for the A11y form with an error exercise.

# Validation
- Open the folder of this repository on your local machine.
- Open the index.html on your browser. It should display the page for the A11y form exercise.
<img width="1784" alt="Screen Shot 2024-02-06 at 5 26 11 PM" src="https://github.com/ct-aa/dotTheory---A11y/assets/93116252/7ac646da-9906-4ebd-a822-a8fba0222cc5">
- Open the error.html on your browser. It should display the page for the A11y form with an error exercise.
<img width="1779" alt="Screen Shot 2024-02-06 at 5 26 17 PM" src="https://github.com/ct-aa/dotTheory---A11y/assets/93116252/cfe9f900-e2a4-423b-bc31-ba46bc735ff1">

# Risks
N/A
